models/Restaurant.ts

```diff
@@ -0,0 +1,39 @@
+import mongoose from "mongoose";
+import Hour from "./Hour";
+import Post from "./Post";
+import FeaturedItem from "./FeaturedItem";
+
+/**
+ * @interface
+ * @typedef {Restaurant} Restaurant Represents a restaurant object interface
+ * @property {string} name Restaurant name
+ * @property {string} bannerPicture Banner picture
+ * @property {string} profilePicture Avatar picture
+ * @property {string} handle Restaurant account handle
+ * @property {string} bio Restaurant bio
+ * @property {string} cuisine Restaurant cuisine type
+ * @property {string} price Restaurant price range
+ * @property {string} address Restaurant address
+ * @property {string} hours Restaurant open hours
```

Comment:
JSDoc includes two properties for hours - not sure if you forgot to delete the string hours?

Reply:
Thank you for pointing it out! I deleted both from jsdoc.